src/providers/ad/ad_gpo.c

```diff
@@ -100,7 +100,6 @@
 /* If INI_PARSE_IGNORE_NON_KVP is not defined, use 0 (no effect) */
 #ifndef INI_PARSE_IGNORE_NON_KVP
 #define INI_PARSE_IGNORE_NON_KVP 0
-#warning INI_PARSE_IGNORE_NON_KVP not defined.
```

Comment:
NACK,
this warning is intentional. It was added because sssd need to be compiled even with libini_config-1.2.
Because required the latest libini_config-1.3 is only in fedora and is required for
https://fedorahosted.org/sssd/ticket/2751